src/main/java/loaders/LocalPdfReader.java

```diff
@@ -11,17 +14,21 @@
 public class LocalPdfReader implements IPdfLoader {
 	private PdfReader reader;
 	private PdfStamper stamper;
-	private String sourceFile;
+	private InputStream sourceStream;
 	
-	public LocalPdfReader(String src) {
-		sourceFile = src;
+	public LocalPdfReader(String srcFilePath) throws FileNotFoundException {
+		sourceStream = new FileInputStream(srcFilePath);
+	}
+
+	public LocalPdfReader(InputStream stream) {
```

Comment:
I added this alternate constructor so the test code can use a classpath resource rather than a filename.